CHANGELOG.md

```diff
@@ -1,4 +1,33 @@
 ## 2.15.0 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-6.0.2210**
+- Supported Terraform version: **v0.12.x**
+
+### Features:
+1. New data sources:
+  - **data_source_aviatrix_firewall**
+  - **data_source_vpc_tracker**
+2. Implemented support for option to manage attachment on either **aviatrix_vpn_profile** or **aviatrix_vpn_user**
+3. Implemented support for ``action`` under domain_names for **aviatrix_fqdn**
+4. Implemented support for adding new users with Geo VPN
```

Comment:
4. Implemented support for adding VPN users under GeoVPN workflow

Reply:
GeoVPN/UDP load balancer

Reply:
2. Implemented support for the option to manage attachment on either **aviatrix_vpn_profile** or **aviatrix_vpn_user** using ``manage_user_attachment`` (and ``profiles`` for the user)
3. Implemented support for ``action`` under the ``domain_names`` filters for **aviatrix_fqdn**